models/linkages/R/read_restart.LINKAGES.R

```diff
@@ -42,30 +42,30 @@ read_restart.LINKAGES <- function(outdir, runid, stop.time, settings, var.names
   forecast <- list()
 
   if ("Fcomp" %in% var.names) {
-    forecast[[1]] <- ens$AGB.pft #already has C  #* unit.conv 
-    names(forecast[[1]]) <- paste0('Fcomp.',pft.names)
+    forecast[[length(forecast)+1]] <- ens$AGB.pft #already has C  #* unit.conv 
+    names(forecast[[length(forecast)+1]]) <- paste0('Fcomp.',pft.names)
```

Comment:
I think all the `names(...)` ones should just be `[[length(forecast)]]` instead of `[[length(forecast)+1]]`

Reply:
@araiho @tonygardella this also still needs to be fixed